src/Castle.Core.Tests/Castle.Core.Tests-VS2017.csproj

```diff
@@ -9,8 +9,12 @@
     <AssemblyOriginatorKeyFile>..\..\buildscripts\CastleKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <EmbeddedResource Include="Core.Tests\Resources\file1.txt" />
-    <EmbeddedResource Include="Core.Tests\Resources\MoreRes\file2.txt" />
+    <EmbeddedResource Include="Core.Tests\Resources\file1.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Core.Tests\Resources\MoreRes\file2.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
They are embedded resources, I'm not sure how copying them to the output directory fixes the problem.

https://github.com/castleproject/Core/blob/master/src/Castle.Core.Tests/project.json#L51-L55